packages/extension-commerce-bm-seo/src/components/seo-hoc.tsx

```diff
@@ -29,18 +29,43 @@ const getComponent = (
     return ComponentClass
 }
 
+const CATCH_ALL_PATH = '*'
```

Comment:
Lets start out on a good food and create a `constants.ts` file in your extension that you can use to place all these.